op-challenger/Makefile

```diff
@@ -9,21 +9,24 @@ LDFLAGS := -ldflags "$(LDFLAGSSTRING)"
 
 op-challenger:
 	env GO111MODULE=on GOOS=$(TARGETOS) GOARCH=$(TARGETARCH) go build -v $(LDFLAGS) -o ./bin/op-challenger ./cmd
+.PHONY: op-challenger
 
 fuzz:
 	go test -run NOTAREALTEST -v -fuzztime 10s -fuzz FuzzKeccak ./game/keccak/matrix
+.PHONY: fuzz
 
 clean:
 	rm bin/op-challenger
+.PHONY: clean
+
```

Comment:
This should already be covered by semgrep across whole codebase no?

Reply:
Yea, it is covered by semgrep right now